sdk/src/connections/usb/macos/usb_depth_sensor_macos.cpp

```diff
@@ -33,8 +33,8 @@
 
 struct UsbDepthSensor::ImplData {};
 
-UsbDepthSensor::UsbDepthSensor(
-    const aditof::DeviceConstructionData & /*data*/) {}
+UsbDepthSensor::UsbDepthSensor(const std::string &driverPath)
+    : m_driverPath(driverPath), m_implData(new UsbDepthSensor::ImplData) {}
```

Comment:
Since you allocate memory for ImplData with new you must delete it at some point. E.g. in the destructor.

Reply:
actually, I've checked and m_implData is a unique_ptr so all is good. Sorry!